src/pages/iou/iouTansactionPropTypes.js

```diff
@@ -0,0 +1,12 @@
+import PropTypes from 'prop-types';
+
+export default {
```

Comment:
Shouldn't this also have a transactionID?

Reply:
Good spot. I was missing `created` too.